### PR TITLE
Fixed issues #1, #3 added some new features.

### PR DESCRIPTION
**Pull request by sachinkumarsingh092**
Added a sticker to welcome new users(see issue #3).
Added 'unknown' handler to reply to unknown commands.
Fixed error in 'empty_message' handle dispatcher.
Fixed errors to handle exception in 'welcome' and 'empty_message'.
Removed unnecessary comments and added meaningful ones in 'welcome' making code more concise(see issue #1).

### CODE REVIEW

File: ilugd.py [hunk 1]
```diff
@@ -82,17 +82,25 @@ def help(bot, update):
 /github - link to ilugd github repos
 ''')
 
+def unknown(bot, update):
```
Comment (xeon-zolt) [inline ilugd.py:85]: nice

File: ilugd.py [hunk 2]
```diff
@@ -82,17 +82,25 @@ def help(bot, update):
...
-               #'Hello {}! Welcome to {} .Please introduce yourself.'.format(message.new_chat_member.first_name,message.chat.title)
-                ]
+
+    try:
+        # new_chat_members return a list of dicts from which member's username has to be searched
+        member_first_name = [member['first_name'] for member in message.new_chat_members if 'first_name' in member]
+    except KeyError:
+        member_first_name = message.new_chat_members[0]['first_name']
+		
+    phrases = ['Hello {}! Welcome to {} .Please introduce yourself. Dont forget to read the chat rules using !rules command'.format(member_first_name,message.chat.title)]
```
Comment (xeon-zolt) [inline ilugd.py]: why modify the list it was planned to expand it so that if multiple people join on same day its too boring to receive same message

Comment (sachinkumarsingh092) [reply ilugd.py]: There was no documentation or comment about it. I will write a comment and add a few more welcoming messages. Should I proceed?

Comment (xeon-zolt) [reply ilugd.py]: applies to same sticker what do you think (i was thinking like a list and throw a random one to chat)

Comment (xeon-zolt) [reply ilugd.py]: > There was no documentation or comment about it. I will write a comment and add a few more welcoming messages. Should I proceed?

yupz (also for stickers (I love gifs))

Comment (sachinkumarsingh092) [reply ilugd.py]: It sounds awesome! Also what about the '!rules' command mentioned in 'phrases'? How should I go about it?

Comment (xeon-zolt) [reply ilugd.py]: ping @nk521 for more info (@nk521 we can merge both ?)
it's from a different bot

Comment (sachinkumarsingh092) [reply ilugd.py]: So, for the time being, I will make changes for stickers and 'phrases' and will raise an issue about the 'rules' command to be solved in future commits. Is that all right?

File: ilugd.py [hunk 3]
```diff
@@ -6,7 +6,7 @@
 from random import choice
 import os
 
-BOTNAME = 'Chutia_bot'
```
Comment (sachinkumarsingh092) [inline ilugd.py]: Sorry, this was added and committed by mistake. I really apologize for it.

**Comment by sachinkumarsingh092** [comment]
I messed up a bit. Please find the latest changes to be merged. Sorry about that.

**Pull request closed without merging**